### ISSUE TITLE
Examples for Pruning with Lpot

### ISSUE THREAD

**Issue by willrnt05** [intel/neural-compressor#18]
Hello, 

I am working on building a front-end focused platform that can handle pruning for the user. I wanted to know if there are any examples in the repository for pruning or if someone could provide an example from their own experiments. I appreciate all your help.

Thank you,

Will

**Comment by zhaoruic-intel** [comment]
Hi Will,

Lpot supports pruning on pytorch backend now. Please find the examples as following:
- magnitude pruning on imagenet: https://github.com/intel/lpot/tree/master/examples/pytorch/eager/image_recognition/imagenet/cpu/prune
- Pruning on bert tasks: we enabled pattern-lock and gradient sensitivity pruning on serverl bert tasks. Please refer to: https://github.com/intel/lpot/blob/master/examples/pytorch/eager/huggingface_models/README.md. 

Thanks,
Zhaorui

**Comment by yutveg** [comment]
Does Lpot support pruning with TensorFlow?

**Comment by ftian1** [comment]
> Does Lpot support pruning with TensorFlow? Are there any examples of this?

not yet. It's still WIP.

**Comment by chensuyue** [comment]
No feed back in 2 weeks, closed at first.
